spring/boot-autoconfigure/src/main/java/com/linecorp/armeria/spring/MeterIdPrefixFunctionFactory.java

```diff
@@ -27,10 +27,20 @@
 
     /**
      * The default {@link MeterIdPrefixFunctionFactory} instance.
+     *
+     * @deprecated Use {@link #ofDefault()}.
      */
-    MeterIdPrefixFunctionFactory DEFAULT = (type, serviceName) ->
-            MeterIdPrefixFunction.ofDefault("armeria." + requireNonNull(type, "type"))
-                                 .withTags("service", requireNonNull(serviceName, "serviceName"));
+    @Deprecated
+    MeterIdPrefixFunctionFactory DEFAULT = ofDefault();
+
+    /**
+     * Returns the default {@link MeterIdPrefixFunctionFactory} instance.
+     */
+    static MeterIdPrefixFunctionFactory ofDefault() {
```

Comment:
This stopped being a singleton - probably not a big issue but just to make sure

Reply:
Extracted into a separate singleton.